ztp/source-cluster-crs/pao/01-pao-subscription.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-performance-addon-operator
+  annotations:
+    workload.openshift.io/allowed: management
+  labels:
+    openshift.io/cluster-monitoring: "true"
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+---
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
I believe this will need to be updated on customer base.

Reply:
lets remove the catalogSource from day0 move it to day2 and keep the OperatorGroup & NS in day0. This will help avoid the race condition issue for operator installation. Will add all other operatorGroups as well SRIOV and Ptp